src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -135,6 +135,8 @@
                               Content="{x:Static local:AdvancedOptionPageStrings.Option_Split_string_literals_on_enter}" />
                     <CheckBox x:Name="Report_invalid_placeholders_in_string_dot_format_calls"
                               Content="{x:Static local:AdvancedOptionPageStrings.Option_Report_invalid_placeholders_in_string_dot_format_calls}" />
+                    <CheckBox x:Name="Enable_all_features_in_opened_files_from_source_generators"
+                              Content="{x:Static local:AdvancedOptionPageStrings.Enable_all_features_in_opened_files_from_source_generators_experimental}" />
```

Comment:
this is odd.  why do we have an option for this.  who are we expecting to disable this?

Reply:
@CyrusNajmabadi This is just a feature flag because to mitigate risk since we're putting this in late for 16.10. We expect to remove this in 16.11 or 17.0.